pyseir/deployment/webui_data_adaptor_v1.py

```diff
@@ -68,7 +72,9 @@ def load_inference_result(self) -> Mapping[str, Any]:
         : dict
             Dictionary of fit result information.
         """
-        return load_inference_result(self.region)
+        if self._mle_fit_result is not None:
+            return self._mle_fit_result
+        return pipeline.load_inference_result(self.region)
```

Comment:
will we ever expect this to be reached?

Reply:
Hopefully not long term. For now it is reached by code that uses the RegionalInput.from_fips constructor, which is called by the non build-all flows.